e2e/test.js

```diff
@@ -117,13 +118,99 @@ async function test() {
   return failed;
 }
 
+
+async function test_from_online_json() {
+
+  let failed = false;
+  for (const country of [ 'GBR' ]) {
+
+        let beta = JSON.parse(
+          fs.readFileSync(`./data/${country}_test_fit.json`)
+        ).beta_set;
+
+        let tt_beta = JSON.parse(
+          fs.readFileSync(`./data/${country}_test_fit.json`)
+        ).tt_beta;
+        
+        let prob_non_severe_death_treatment = JSON.parse(
+          fs.readFileSync(`./data/${country}.json`)
+        ).prob_non_severe_death_treatment;
+
+        let prob_severe_death_treatment = JSON.parse(
+          fs.readFileSync(`./data/${country}.json`)
+        ).prob_severe_death_treatment;
+
+        let hosp_beds = JSON.parse(
+          fs.readFileSync(`./data/${country}_test_fit.json`)
+        ).hosp_beds;
+
+        let ICU_beds = JSON.parse(
+          fs.readFileSync(`./data/${country}_test_fit.json`)
+        ).ICU_beds;
```

Comment:
Geek moment: Javascript has this lovely syntax when your variable names match your assignment names. It's called [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

```suggestion
        let {
          beta,
          tt_beta,
          hosp_beds,
          ICU_beds
        } = JSON.parse(fs.readFileSync(`./data/${country}_test_fit.json`));
        
        let {
          prob_non_severe_death_treatment,
          prob_severe_death_treatment
        } = JSON.parse(fs.readFileSync(`./data/${country}.json`))
```

Reply:
Huh very cool - thanks for showing me this. Learn js has been on my yearly goal lists for .... a while. So I will commit this to memory and this can be the first year I can honestly say I've made a step towards it.